lib/packwerk/run_context.rb

```diff
@@ -119,7 +124,7 @@ def checkers
     def constant_name_inspectors
       [
         ::Packwerk::ConstNodeInspector.new,
-        ::Packwerk::AssociationInspector.new(inflector: @inflector, custom_associations: @custom_associations),
+        ::Packwerk::AssociationInspector.new(inflector: inflector, custom_associations: custom_associations),
```

Comment:
I agree we should do this, but there are still references to `@load_paths`, `@package_paths`, `@inflector` and `@checker_classes` throughout the class — should we clean those up too?